Formula/glooctl.rb

```diff
@@ -2,15 +2,15 @@ class Glooctl < Formula
   desc "Envoy-Powered API Gateway"
   homepage "https://gloo.solo.io"
   url "https://github.com/solo-io/gloo.git",
-      :tag      => "v0.19.1",
-      :revision => "1539067a393f53a0b793106aaa4676f8ff87df99"
+      :tag      => "v0.20.1",
+      :revision => "102a3a56c1b2059c2177a2e1629d52d8dc5b0639"
   head "https://github.com/solo-io/gloo.git"
 
   bottle do
     cellar :any_skip_relocation
-    sha256 "2887e746e434e1ffa1e58b76217bdde6872f3f2b487a628d5d5032d573489e66" => :mojave
-    sha256 "ca1da47eac784176de1a4bd9713b543e7868385331ff3a4b54a77cd7fcaac393" => :high_sierra
-    sha256 "a4cfffa96ebb809203e5119ae4bdaed47300e90fb7272236b20d0f186c14e42e" => :sierra
+    sha256 "d94a15551f69a953edd7ef7efffd5769e1038c371d020c4c7bc0eda8d90433aa" => :catalina
```

Comment:
The bottle block should not be modified, it is handled automatically (from the binaries created on our CI) when we merge

Reply:
This is a misleading diff. There is no difference between this part of the file and the contents of https://github.com/Homebrew/homebrew-core/blob/master/Formula/glooctl.rb#L9-L14